data/md/legislature/Jon-S-Cardin-48959ab9-e2d5-4928-840d-bd0ca5d86696.yml

```diff
@@ -0,0 +1,13 @@
+id: ocd-person/48959ab9-e2d5-4928-840d-bd0ca5d86696
+name: Jon S. Cardin
+image: http://mgaleg.maryland.gov/2023RS/images/cardin01.jpg
+party:
+- name: Democratic
+roles:
```

Comment:
@sroomf this person should already have a file elsewhere and we just need to add:
- type: lower
  jurisdiction: ocd-jurisdiction/country:us/state:md/government
  district: 11B
They previously were in house district 11 from Jan 2019 until 01/11/2023